### DISCUSSION TITLE
How to set gradient and background image from Astro import?

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#11003] [Help]
This is my current setup:
```jsx
import flag from "../assets/flag.jpeg";
const heroStyle = `background-image: linear-gradient(to bottom, rgba(0,0,0,0), rgba(38,38,38,0.8), rgba(38,38,38,1)), url('${flag.src}');`;
---

<div class="grid place-items-center text-center bg-cover bg-no-repeat mb-10" style={heroStyle}>
...
</div>
```

It's definitely a mess but I'm not sure what tailwind class to use in order to get a gradient atop an image, which is added via Astro import. So then there are two facets: 

1. How do I use Tailwind to accomplish a background image and background gradient simultaneously, and
2. How would I then use the imported image `flag.src` in the background image class? 

Thanks in advance.

**Comment by wongjn** [comment]
1. Use an arbitrary class or a custom `backgroundImage` value in the Tailwind configuration.
2. You wouldn't use it directly. Either use the `style` attribute as you're doing now, or set the import path variable to a CSS variable and then use the CSS variable in the Tailwind class.
